Content.Shared/Damage/Components/ClothingSlowOnDamageModifierComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Damage.Components;
+
+/// <summary>
+/// This is used for a clothing item that modifies the slowdown from taking damage.
```

Comment:
Can't find anything wrong with the code, it looks solid to me. I recommend adding a `<see cref="SlowOnDamageComponent"/>` reference to the component's summary though, just to make it explicitly clear what values it is modifying.